examples/dh-5-example/README.md

```diff
@@ -0,0 +1,27 @@
+This example helps Getting Started with using and configuring ingestion, mapping and mastering flows.
+There are 3 flow artifacts available in the flows directory.
+Before starting with this example, run the following gradle task to extract the input files directory from input.zip file
+./gradlew extractZip
+
+1) ingestion_only-flow.flow.json - This file contains 7 steps. Each step refers to ingesting different format of input files. The input files are located in the "input" directory
+
+2) ingestion_mapping-flow.flow.json - This file contains 4 steps. The input files are located in the "input" directory
+	Step 1 and Step 2 to ingest and harmonize (using mapping) json documents.
+	Step 3 and Step 4 to ingest and harmonize (using mapping) xml documents.
+
+3) ingestion_mapping_mastering-flow.flow.json - This file contains 3 steps. The input files are located in the "mastering-input" directory
+	Step 1 to ingest json documents
+	Step 2 to harmonize json documents
+	Step 3 to master json documents
+
+
+gradle task to run a Flow:
+
+1) ./gradlew hubRunFlow -PflowName=ingestion_only-flow

```

Comment:
Before running a flow, need to tell the user to modify gradle-local.properties to include their username/password, and then to run "mlDeploy" to deploy the application.

Reply:
Updated the README